[2023-07-10T17:13:38.382+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_imports_v11.get_data manual__2023-07-10T17:13:37.405949+00:00 [queued]>
[2023-07-10T17:13:38.388+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_imports_v11.get_data manual__2023-07-10T17:13:37.405949+00:00 [queued]>
[2023-07-10T17:13:38.388+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-10T17:13:38.398+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): get_data> on 2023-07-10 17:13:37.405949+00:00
[2023-07-10T17:13:38.402+0000] {standard_task_runner.py:57} INFO - Started process 3814 to run task
[2023-07-10T17:13:38.405+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'test_imports_v11', 'get_data', 'manual__2023-07-10T17:13:37.405949+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/scrap_data.py', '--cfg-path', '/tmp/tmppz6oiyvx']
[2023-07-10T17:13:38.405+0000] {standard_task_runner.py:85} INFO - Job 179: Subtask get_data
[2023-07-10T17:13:38.435+0000] {task_command.py:410} INFO - Running <TaskInstance: test_imports_v11.get_data manual__2023-07-10T17:13:37.405949+00:00 [running]> on host 3371e7602d27
[2023-07-10T17:13:38.502+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_imports_v11' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-10T17:13:37.405949+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-10T17:13:37.405949+00:00'
[2023-07-10T17:13:38.503+0000] {logging_mixin.py:149} INFO - /usr/local/bin/python
[2023-07-10T17:13:38.503+0000] {logging_mixin.py:149} INFO - ['/home/***/.local/bin', '/usr/local/lib/python37.zip', '/usr/local/lib/python3.7', '/usr/local/lib/python3.7/lib-dynload', '/home/***/.local/lib/python3.7/site-packages', '/usr/local/lib/python3.7/site-packages', '/opt/***/dags', '/opt/***/config', '/opt/***/plugins']
[2023-07-10T17:13:38.503+0000] {logging_mixin.py:149} INFO - 4.10.0
[2023-07-10T17:13:38.759+0000] {logging_mixin.py:149} INFO - 1.3.5
[2023-07-10T17:13:38.759+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scrap_data.py", line 32, in get_data
    import pyspark
ModuleNotFoundError: No module named 'pyspark'
[2023-07-10T17:13:38.765+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=test_imports_v11, task_id=get_data, execution_date=20230710T171337, start_date=20230710T171338, end_date=20230710T171338
[2023-07-10T17:13:38.777+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 179 for task get_data (No module named 'pyspark'; 3814)
[2023-07-10T17:13:38.817+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-10T17:13:38.832+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
